internal/dao/sqlx/source_dao.go

```diff
@@ -0,0 +1,95 @@
+package sqlx
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	"github.com/RHEnVision/provisioning-backend/internal/db"
+	m "github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/jmoiron/sqlx"
+)
+
+const (
+	create     = `INSERT INTO sources (source_id, name, org_id, auth_id) VALUES ($1, $2, $3, $4) RETURNING created_at`
```

Comment:
At any time you can opt-in to sqlx.NamedStatement where you can use `:this, :style, :of, :arguments` but I am fine with this. Anything up to let's say five is still readable.

It is typically good idea to also put ID to RETURNING clause so you also get ID as well so the value is returned back to the payload with ID.

Reply:
> At any time you can opt-in to sqlx.NamedStatement where you can use `:this, :style, :of, :arguments` but I am fine with this. Anything up to let's say five is still readable.

Can you give me an example? I am not sure I understood

Reply:
Its okay no need for a change, just for your info: http://jmoiron.github.io/sqlx/#namedParams